packages/web3-validator/src/validation/string.ts

```diff
@@ -25,6 +25,22 @@ export const isString = (value: ValidInputTypes) => typeof value === 'string';
 export const isHexStrict = (hex: ValidInputTypes) =>
 	typeof hex === 'string' && /^((-)?0x[0-9a-f]+|(0x))$/i.test(hex);
 
+/**
+ * Is the string a hex string.
+ *
+ * @param  value
+ * @param  length
+ * @returns  output the string is a hex string
+ */
+export function isHexString(value: string, length?: number): boolean {
```

Comment:
We should also move some tests for these functions ( after this PR may be ) so we dnt drop coverage. With this PR its dropped 9% and we are below 80% goal. https://github.com/web3/web3.js/issues/5973

Reply:
yes tests. is the next my step